src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -258,14 +261,28 @@ public void findByUserIdOrAndSinceDate(String sinceDate, String userId) {
             assertThat(testResults.get("errorDescription")).isEqualTo("001 missing/invalid parameter");
         } else {
             List<HashMap> users = (List<HashMap>) testResults.get("users");
-            lastUpdateTime = (String) users.get(0).get("lastUpdated");
+
+            String updateDateTime = (String) users.get(0).get("lastUpdated");
+            lastUpdateTime = formatDate(updateDateTime);
             lastUpdateTimes.add(lastUpdateTime);
             lastRecordIdInPage = (String) testResults.get("lastRecordInPage");
             validateRetrievedUsersDetails(testResults, null);
         }
         log.info("findByUserIdOrAndSinceDate :: END");
     }
 
+    public static String formatDate(String dateString) {
```

Comment:
all good, but can you just make sure all methods in this class have the right access? ie. seem to be lots here which should be private